.github/workflows/security.yaml

```
@@ -90,3 +90,42 @@ jobs:
         uses: github/codeql-action/upload-sarif@v3
         with:
           sarif_file: results.sarif
+
+  # We need to render _every_ directory that has a kustomization.yaml file,
+  # since that's what infra-deployments checks for.  Having a check identical
+  # to what infra-deployments does will save us some embarassment when we make
+  # a release.
+  kubelinter:
+    name: Kubelinter
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v4
+
+      - name: Render kustomize templates
+        run: |
+          mkdir out
+          find operator server -name 'kustomization.yaml' | \
+            xargs -I {} -n1 -P8 \
+            bash -c 'dir=$(dirname "{}"); output_file=$(echo $dir | tr / -)-kustomization.yaml; if ! log=$(kustomize build "$dir" -o "out/$output_file" 2>&1); then echo "Error when running kustomize build for $dir: $log" && exit 1;fi'
```

what you think about overwriting here the image tag and revert the change in default kustomization files? I'd prefer to keep kustomization files as stock as possible.

```suggestion
            bash -c 'dir=$(dirname "{}"); output_file=$(echo $dir | tr / -)-kustomization.yaml; if ! log=$(cd "$dir" && kustomize edit set image workspaces/rest-api:index workspaces/controller:index && kustomize build . -o "out/$output_file" 2>&1); then echo "Error when running kustomize build for $dir: $log" && exit 1;fi'
```

---

or else we may use `make -C operator  package` and `make -C server package` to build the manifests and lint those

---

I pulled this line straight from infra-deployments, and I'd like to keep this in line with them as much as possible.  That said, this _should_ work.

---

I'll also revert things to be back to `controller:latest` and `rest-api:latest` in the deployments.

---

Fixed in d861b91.